client/app/hosting/database/dump/DUMPS.html

```diff
@@ -7,92 +7,43 @@
-                            </button>
-                            <button class="btn btn-icon" type="button"
-                                    data-ng-attr-title="{{tr('database_tabs_dumps_restore_dump_with_creation_date', [(dump.creationDate | date:'medium')] )}}"
-                                    data-ng-click="setAction('database/restore/hosting-database-restore', {bdd:bdd, dump:dump})"
-                                    data-ng-disabled="bdd.waitRestore || dump.waitDelete">
-                                <span class="fa fa-repeat" aria-hidden="true"></span>
-                            </button>
-                            <button class="btn btn-icon" type="button"
-                                    data-ng-attr-title="{{tr('database_tabs_dumps_delete_dump', [(dump.creationDate | date:'medium')] )}}"
-                                    data-ng-click="setAction('database/dump/delete/hosting-database-dump-delete', {database:bdd, dump:dump})"
-                                    data-ng-disabled="dump.waitDelete">
-                                <span class="fa fa-trash" aria-hidden="true"></span>
-                            </button>
-                        </td>
-                    </tr>
-                    <tr data-ng-if="bdd.waitDump">
-                        <td class="text-center" colspan="4">
-                            <span class="label label-info"
-                                  data-i18n-static="database_tabs_dumps_dump_in_progress"></span>
-                        </td>
-                    </tr>
-                </tbody>
-            </table>
-        </div>
-    </div>
-
-    <div class="clearfix"
-         data-ng-if="ctrlDumps.dumpsIds">
-        <div data-pagination-front
-             data-items="ctrlDumps.dumpsIds"
-             data-paginated-items="ctrlDumps.dumpsDetails"
-             data-current-page="currentPage"
-             data-items-per-page="itemsPerPage"
-             data-nb-pages="nbPages"
-             data-transform-item="ctrlDumps.transformItem(item)"
-             data-on-transform-item-done="ctrlDumps.onTransformItemDone(items)"
-             data-page-placeholder="{{tr('pagination_page', [currentPage, nbPages])}}"
-             data-item-per-page-placeholder="{{tr('pagination_display')}}"
-             data-on-page-change="ctrlDumps.dumpsLoading = true"
-             data-ng-show="!ctrlDumps.dumpsLoading && ctrlDumps.dumpsDetails.length > 0">
-        </div>
+    <div>
+        <span data-i18n-static="database_tabs_dumps_change"></span>
+        <strong data-ng-bind="bdd.name"></strong>
     </div>
+    <oui-datagrid data-rows="ctrlDumps.databaseDumps"
+                  data-row-loader="ctrlDumps.transformItem($row)">
+        <oui-column data-title="tr('database_tabs_dumps_snapshot_date')" data-property="snapshotDate">
+            <span data-ng-bind="$row.snapshotDate | date:'mediumDate'"></span>
```

Comment:
That's not your fault but I don't know why there's different filter for date : 'medium' and 'mediumDate'

Reply:
There's a distinction between the two, ```medium``` display date and time and ```mediumDate``` only the date